dev/reference/sql/statements/admin.md

```diff
@@ -70,7 +70,8 @@ mysql> admin show ddl jobs;
 - `ADMIN CANCEL DDL JOBS job_id [, job_id] ...`: To cancel the currently running DDL jobs and return whether the corresponding jobs are successfully cancelled. If the operation fails to cancel the jobs, specific reasons are displayed.
 
     > **Note:**
-    >
+
```

Comment:
```suggestion
    >
```
You need to add `>` for adding a blank line in a note. Otherwise, this note will be split into two notes.